src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int RANDOM_BOUND = 30;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int numberOfFigures = random.nextInt(FIGURE_COUNT - 1); //because from 0 to 4

```

Comment:
```suggestion
        int numberOfFigures = random.nextInt(FIGURE_COUNT); //because from 0 to 4
```
read documentation to `nextInt() `method